src/qt/pivx/sendconfirmdialog.cpp

```diff
@@ -123,6 +123,16 @@ void TxDetailDialog::setData(WalletModel *_model, const QModelIndex &index)
         ui->textStatus->setText(QString::fromStdString(rec->statusToString()));
         ui->textSize->setText(QString::number(rec->size) + " bytes");
 
+        // If there is a memo in this record
+        if (rec->memo) {
+            ui->textMemo->setText(QString::fromStdString(*rec->memo));
+            ui->contentMemo->setVisible(true);
```

Comment:
add this to enable word-wrap so longer memo messages don't clip out the right side:
```cpp
ui->textMemo->setWordWrap(true);
```
-------------
Before: 
![ScreenShot_20201201011616](https://user-images.githubusercontent.com/7393257/100721732-8f664800-3374-11eb-9683-2f9e851d6fc4.png)

After:
![ScreenShot_20201201012826](https://user-images.githubusercontent.com/7393257/100721749-968d5600-3374-11eb-8954-cd9269e38762.png)

Reply:
Looks good 👍

Reply:
yep good. Will add it directly into the .ui file instead of set it programmatically .